test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/DropInvalidMetadataTests.cs

```diff
@@ -33,6 +33,7 @@ public DropInvalidMetadataTests(EnableDropInvalidDicomJsonMetadataHttpIntegratio
     }
 
     [Fact]
+    [Trait("Category", "bvt-leniency")]
```

Comment:
Why do we need this to be bvt? Is'nt OSS validation enough?

Reply:
How is this different from running in OSS vs PaaS? PaaS tests test bvt like 1 STOW, WADO and QIDO. Or anything that is different in PaaS. This does not qualify either. It can just stay in OSS